unix/configure.ac

```diff
@@ -728,6 +728,9 @@ if test x"$C99_COMPATIBLE_RADIOSITY" != x"" || test x"$ac_cv_sizeof_int" != x"4"
   fi
 fi
 
+AC_HEADER_TIOCGWINSZ
```

Comment:
It seems a bit fishy to me that you're using the `AC_HEADER_TIOCGWINSZ` in the autoconf script, but not testing `GWINSZ_IN_SYS_IOCTL` in the C++ code. Please double-check.